my-profile-webapp/src/main/webapp/my-app/lec/partials/emergency-phone.html

```diff
@@ -0,0 +1,41 @@
+<loading-gif data-object='emergencyPhoneNumbers' data-reuse="true"></loading-gif>
+<div class='info-card' ng-if="empty || emergencyPhoneNumbers.emergencyPhoneNumbers.length>0">
+  <h1>My Emergency Phone Number</h1>
+  <div class="description-box">
+    <p>An emergency phone number where you can be reached in cases of safety</p>
```

Comment:
Did some research on the interwebs for this and found that this phrase was used commonly in this situation. I've already talked with @keirserrie and he's going to help with some of this user experience things.  I'm definitely open to redoing this whole UI/messaging.  

Added @keirserrie to comment so he could see @apetro suggestion.